packages/react-widgets/src/models/CategoryModel.js

```diff
@@ -34,9 +29,7 @@ export const getCategories = async (props) => {
   const operationColumn = props.operationColumn || column;
 
   if (viewportFilter) {
-    return filterViewportFeaturesToGetCategories({
```

Comment:
I'm not sure if we should force the use of workers here in the models. It feels like we're losing a path to calculate with geojson features in the near-future

Reply:
We already decided on this